manifests/x/Xanthus58/VanillaRenewed/2.3.0/Xanthus58.VanillaRenewed.locale.en-US.yaml

```
@@ -0,0 +1,10 @@
+PackageIdentifier: Xanthus58.VanillaRenewed
+PackageVersion: 2.3.0
+PackageLocale: en-US
+Publisher: Xanthus58
+PackageName: Vanilla Renewed
```

```suggestion
PackageName: VanillaRenewed
```
to match Control Panel.